lintly/parsers.py

```diff
@@ -234,6 +234,57 @@ def parse_violations(self, output):
         return violations
 
 
+class BanditJSONParser(BaseLintParser):
+    """
+    Bandit JSON format:
+
+      [
```

Comment:
Since the input expects a *full* bandit json file and you're indexing into results, this format should reflect that.
```
    Bandit JSON format:

    {
      "errors": [],
      "generated_at": "2021-01-07T23:39:39Z",
      "metrics": {
        ...
      }
      "results": [
        {
          "code": "13 \n14 env = Environment(\n ...",
          "filename": "./lintly/formatters.py",
          "issue_confidence": "HIGH",
          "issue_severity": "HIGH",
          "issue_text": "Using jinja2 templates with autoescape=False is dangerous ...",
          "line_number": 14,
          "line_range": [
            14,
            15,
            16
          ],
          "more_info": "https://bandit.readthedocs.io/en/latest/plugins/b701_jinja2_autoescape_false.html",
          "test_id": "B701",
          "test_name": "jinja2_autoescape_false"
        },
        ...
      ]
    }
```

Reply:
Also please fix the indenting overall.

Reply:
Yes I will add this.